### ISSUE TITLE
CYBERSHOT DSC-TX5 "no tweaks available"

### ISSUE THREAD

**Issue by valdybruv** [ma1co/Sony-PMCA-RE#484]
Hi there,

Is anyone able to help with a tweak or firmware file that I can use to unlock English on my Japanese model DSC-TX5?
It would be greatly appreciated.

Thank you so much!

**Comment by 9lqw** [comment]
I'm looking for a solution as well... my console says "no tweaks available"

**Comment by dreambold** [comment]
@9lqw Did you try it on Ubuntu?

**Comment by 9lqw** [comment]
> [@9lqw](https://github.com/9lqw) Did you try it on Ubuntu?

Yes, ubuntu 22.04 (jammy)

**Comment by dreambold** [comment]
@9lqw It should be working as it's in the supported camera list.

![Image](https://github.com/user-attachments/assets/80acdc08-0890-4c63-a444-9e6f95409f40)
Do you have anydesk? I can help you with the tweaks.
Updatershell and Serviceshell should work

**Comment by 9lqw** [comment]
The command for updatershell/ serviceshell did run, but Serviceshell says that there are no tweaks available for the camera. Do you have the tweaks for this specific camera? Or perhaps there's an override?

**Comment by dreambold** [comment]
Did you set the usb connection to MASS STORAGE in the camera menu?

**Comment by 9lqw** [comment]
Yep, otherwise it wouldn't connect at all

**Comment by dreambold** [comment]
Did you try to run the command with SUDO?

**Comment by 9lqw** [comment]
Yeah, i tried both the command-line as well as the GUI app with sudo or sudo su permissions, both say "no tweaks available" after validating that the camera is in service mode

**Comment by dreambold** [comment]
If you have anydesk, I can try to help you by a remote session.

**Comment by 9lqw** [comment]
i'd be down to install it if you're willing to do that ☃️ just let me know when

**Comment by dreambold** [comment]
Sure, let me know after you install anydesk.

**Comment by 9lqw** [comment]
Hey, Sorry for the late reply, I had exams this week. I'm free now if you are

**Comment by dreambold** [comment]
Okay, here's my teams https://teams.live.com/l/invite/FBAkakI9xj3CZ7zxwc, let's chat there
